nautobot/core/management/commands/generate_performance_test_endpoints.py

```diff
@@ -14,6 +29,13 @@ class Command(BaseCommand):
 
     help = "List all relevant performance test url patterns in Nautobot Core"
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--output-file",
+            default=False,
```

Comment:
More usual pattern would be to let the caller specify a path to output to, rather than this being a simple boolean flag?